EIPS/eip-3009.md

```diff
@@ -0,0 +1,317 @@
+---
+eip: 3009
+title: `transferWithAuthorization` - Gas-Abstracted ERC20 transactions with EIP-712
+author: Peter Jihoon Kim (@petejkim), Kevin Britz (@kbrizzle), David Knott (@DavidLKnott)
+discussions-to: https://github.com/ethereum/EIPs/issues/3010
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-09-28
+requires: 20, 712
+---
+
+## Simple Summary
+
+A function called `transferWithAuthorization` to enable gas-abstracted and atomic interactions with [ERC-20](https://eips.ethereum.org/EIPS/eip-20) token contracts via signatures conforming to the [EIP-712](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md) typed message signing specification.
+
+This enables the user to:
+* delegate the gas payment to someone else,
+* pay for gas in the token itself rather than in ETH,
+* perform one or more token transfers and other operations in a single atomic transaction,
+* transfer ERC-20 tokens to another address, and have the recipient submit the transaction,
+* batch multiple transactions with minimal overhead, and
+* create and perform multiple transactions without having to worry about them failing due to accidental nonce-reuse or improper ordering by the miner.
+
+The popular USD-backed stablecoin [USDC v2](https://etherscan.io/token/0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48) implements an expanded form of this spec. This can also be adopted alongside the [EIP-2612](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2612.md) spec for maximum compatibility with existing applications.
+
+## Abstract
+
+This is a standard that extends the ERC-20 spec by introducing a new function, `transferWithAuthorization`. This function enables users to transfer tokens without also having to submit a transaction and pay for gas themselves. This function also can also be used to transfer tokens and perform another action in a single atomic transaction.
+
+## Motivation
+
+There is an existing spec, [EIP-2612](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2612.md), that also allows gas-abstracted transactions. The two primary differences between this spec and EIP-2612 are that:
```

Comment:
Also, note that "meta-transaction" is generally the preferred terminology for transaction which are authorize by signatures.

Reply:
Agreed, changed. Thank you.